### ISSUE TITLE
Cannot retrieve only the ID property from DB

### ISSUE THREAD

**Issue by Mihai-B** [isar/isar#391]
**Steps to Reproduce**

Trying to get only the ids property from the database results in an error like:
```
E/flutter (16051): [ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: IsarError: Unknown propery "id"
E/flutter (16051): #0      CollectionSchema.propertyIdOrErr
package:isar/src/collection_schema.dart:123
E/flutter (16051): #1      buildNativeQuery
package:isar/…/native/query_build.dart:80
E/flutter (16051): #2      IsarCollectionImpl.buildQuery
package:isar/…/native/isar_collection_impl.dart:480
E/flutter (16051): #3      QueryBuilder.buildInternal
package:isar/src/query_builder.dart:252
E/flutter (16051): #4      QueryExecute.build
```

**Code sample**
I have declared the entity as so:
```
abstract class BaseBuilding extends Equatable {
  @Id()
  final int id;
  final String abcd;

  const BaseBuilding({
    required this.id,
    required this.abcd,
  });

  @override
  bool get stringify => true;

  @override
  List<Object> get props => [abcd, id];
}

Collection(inheritance: true, accessor: "apartment")
class Apartment extends BaseBuilding {
  const Apartment({
    required super.id,
    required super.abcd,
  });
}
```

And then I try to run this query:
```
final items = isarConnection.writeTxnSync((conn) {
   final appartments = conn.apartment
          .where()
          .anyId()
          .idProperty()
          .build()
          .findAllSync();

    ....
}

```
Is this a bug or am I doing something wrong here ?

**Version**
 - Platform: Android,
 - Flutter version: 3.0.1
 - Isar version: 2.5.0
 - Isar Generator version: 2.5.0

**Comment by leisim** [comment]
It's a bug. Thanks for reporting 👍

**Comment by leisim** [comment]
Fixed in `3.0.0-dev.2`
